cvat/apps/lambda_manager/views.py

```diff
@@ -599,6 +667,96 @@ def reset(self):
                 # FIXME: need to provide the correct version here
                 self.data = {"version": 0, "tags": [], "shapes": [], "tracks": []}
 
+        def parse_anno(anno, labels):
+            label = labels.get(anno["label"])
+            if label is None:
+                # Invalid label provided
+                return None
+
+            if anno.get('attributes'):
+                attrs = [{
+                    'spec_id': label['attributes'][attr['name']],
+                    'value': attr['value']
+                } for attr in anno.get('attributes') if attr['name'] in label['attributes']]
+            else:
+                attrs = []
+
+            if anno["type"].lower() == "tag":
+                return {
+                    "frame": frame,
+                    "label_id": label['id'],
+                    "source": "auto",
+                    "attributes": attrs,
```

Comment:
It would be nice to factor out the properties that are the same in both branches.

Reply:
Now I would not touch it, because we do not have even any model returning "tags" to debug changes